randovania/games/fusion/logic_database/Main Deck.txt

```diff
@@ -1109,7 +1111,9 @@ Extra - room_id: [38]
   * Open Passage to The Attic/Other to Arachnus Arena
   * Extra - door_idx: (200,)
   > Beside Door
-      Missiles
```

Comment:
can you briefly explain what this is all doing for my awareness.

Reply:
Going to arachnus: makes this not a double unsafe action (first generic node you loose access to the door, going down makes you loose access to the generic node). it boosts arachnnus' weight from ~0.3 to1.2 or something
e + attic to generic node: nothing really. just makes it clear that the reason these are currently trivial and not WJ beginner is because of the generator seeing these as unsafe actions otherwise.